<?xml version="1.0"?>
<launch>

<!--
  <node pkg="package_name" type="node_name" name="override_name">
    
  </node>

<include file="$(find package_name)/directory/another.launch"/>
-->

<node pkg="gpr_gps_converge" type="free_converge" name="free_converge" respawn="true">
	<param name="data_folder" value="/home/administrator/sensors_ws/src/gpr_gps_converge/data/"/>
  </node>

</launch>
